test/src/edu/stanford/nlp/process/StopListTest.java

```diff
@@ -0,0 +1,33 @@
+package edu.stanford.nlp.process;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import edu.stanford.nlp.ling.Word;
+
+public class StopListTest {
+
+	@Test
+	public void testGenericWordConstructor() {
+		// Tests that no exception being thrown during instantiation
+		StopList sut = new StopList();
+	}
+	@Test
+	public void testGenericWordConstainsString() {
+		// Tests that a common stopword is contained in the list as string
+		StopList sut = new StopList();
+		boolean containsWord = sut.contains("and");
```

Comment:
What's the reason this isn't just one line, assertTrue(sut.contains("and"))

Reply:
For readability, but I agree it seems like overkill here.
We will replace with an assertion message and condense to one line which should still yield under 80 characters.

Reply:
For readability, but I agree it seems like overkill here.
I replaced it with an assertion message and condensed to one line which should still yield under 80 characters.

Reply:
In that case, do you mind squashing that on your end?  It would make it easier to cherrypick one change at a time.

Reply:
Done! We squashed all these latest changes fixes together into one commit! Also we rebased it from your master.
Please tell us if you want them squashed into even larger/fewer commtis.